misc/windows-gaming.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# This needs to be located in /usr/lib/systemd/system-sleep/
+
+if [ "$1" = "pre" ]; then
+  nc=$(command -v nc)
+  if [ -z "$nc" ]; then
+    nc=$(command -v netcat)
+  fi
+  if [ -z "$nc" ]; then
+    nc=$(command -v ncat)
+  fi
+  if [ -z "$nc" ]; then
+    echo "No netcat, nc or ncat found. Exiting..."
+    exit 1
+  fi
+  # Currently on arch netcat has a bug where it does not end the connection.
+  # Therefore we use a timeout of 3 seconds.
+  echo -ne "\x05" | $nc -w 3 -U /run/user/1000/windows-gaming-driver/control.sock
```

Comment:
The solution for this is the `-N` flag as the server will obviously keep waiting for additional commands until the client closes the connection. (I believe the default behavior of arch's openbsd-netcat changed in a recent update so we now have to specify it but previously didn't.)